rust/main/agents/scraper/src/db/generated/gas_payment.rs

```diff
@@ -33,6 +37,10 @@ pub enum Column {
     GasAmount,
     TxId,
     LogIndex,
+    Origin,
+    Destination,
+    InterchainGasPaymaster,
```

Comment:
we're currently only indexing IGP payments made to the Abacus Works IGP contract (so there's a 1:1 relationship between the domain and the IGP address), but I'm assuming we're storing the IGP address in case we later want to index other non-AW IGP contracts as well?

Reply:
I did not know about 1:1 relationship between domain and IGP. I am using IGP address to be able to retrieve a payment by sequence. Presumably, if we decide to re-deploy IGP and restart sequence, we'll need the IGP address to uniquely identify payment.